out/simple_desc.json

```diff
@@ -1,4 +1,5 @@
 {
+    "version": 1,
```

Comment:
Maybe use a string as version to make it more readable, for example "20201226"

Reply:
I think a number make compare logic more clear.